packages/lesswrong/server/recombee/client.ts

```diff
@@ -495,11 +499,13 @@ const recombeeApi = {
     const { curatedPostIds, stickiedPostIds, excludedPostFilter } = await helpers.getOnsitePostInfo(lwAlgoSettings, context);
 
     const curatedPostReadStatuses = await helpers.getCuratedPostsReadStatuses(lwAlgoSettings, curatedPostIds, userId, context);
+    const manuallyStickiedPostReadStatuses = await helpers.getManuallyStickiedPostsReadStatuses(stickiedPostIds, userId, context);
 
     const includedCuratedPostIds = curatedPostIds.filter(id => !curatedPostReadStatuses.find(readStatus => readStatus.postId === id));
+    const includedManuallyStickiedPostIds = stickiedPostIds.filter(id => !manuallyStickiedPostReadStatuses.find(readStatus => readStatus.postId === id))
```

Comment:
This should be `includedStickiedPostIds`, since in fact it only ever filters out manually stickied posts, not regular stickied posts.